src/components/audio-player/audio-player.stories.js

```diff
@@ -0,0 +1,155 @@
+import html from "../../../.storybook/helpers/html";
+import { useEffect } from "@storybook/client-api";
+import { withKnobs, text, boolean } from "@storybook/addon-knobs";
+import initializeAudioPlayers from "./audio-player.js";
+import greekHall1x1 from "../../../.storybook/assets/images/greek-hall/1x1";
+
+import { playIcon, pauseIcon, rewindTenSecondsIcon, forwardTenSecondsIcon, upCarrotIcon } from "./helper-icons/helper-icons.js";
+
+export default { title: "Media/Audio Player", decorators: [withKnobs] };
+
+const data = () => {
+	return {
+		darkMode: boolean("Dark Mode", false),
+		track: {
+			hasImage: boolean ("Has Image", true),
+			audioFileURL:
+				text("Audio File URL", "https://cdn.bitdegree.org/learn/I_Cactus_-_05_-_ruby_cactus.mp3?raw=true"),
+			coverImage: greekHall1x1,
+			subtitle: text("Subtitle", "Praise Songs about Javascript"),
+			title: text("Title", "Track 1. Title"),
+			transcript: text(
+				"Transcript",
+				"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpaqui officia deserunt mollit anim id est laborum."
+			),
+		}
+	};
+};
+
+const audioPlayerMarkUp = (model, playerMode = "") => {
+	const isDark = model.darkMode ? "inverted-colors" : "";
+
+	return html`
+		<div style="padding: 40px; background-color: #eee">
+			<section class="audio-player js-marble-audio-player ${playerMode} ${isDark}">
+				<div class="audio-player__media-section">
+					<div class="audio-player__image-section">
+						${model.track.hasImage ? html`<div class="audio-player__image-wrapper">
+								<img
+									class="audio-player__cover-image"
+									alt="${model.track.coverImage.alt}"
+									width="${model.track.coverImage.width}"
+									height="${model.track.coverImage.height}"
+									src="${model.track.coverImage.srcSet.fallback}"
+								/>
+							</div>` : ""}
+
+					</div>
+
+					<div class="audio-player__body">
+						<div class="audio-player__headings">
+							<h1 class="audio-player__title">${model.track.title}</h1>
+							<h2 class="audio-player__sub-title">${model.track.subtitle}</h2>
+						</div>
+						<div class="audio-player__controls-wrapper">
+							<div class="audio-controls">
+								<div class="audio-controls__play-wrapper">
+									<button class="js-audio-player__play audio-controls__play">
+										<span class="js-play-icon audio-controls__play-icon"> ${playIcon()} </span>
```

Comment:
@JosephCurley just fyi, this button is not yet accessible.

Perhaps a `<label>` or interior span with a `.screen-reader-only` would suffice. 

<img width="1920" alt="Screen Shot 2021-07-02 at 11 37 23 AM" src="https://user-images.githubusercontent.com/6576730/124298222-01c92180-db2a-11eb-9851-7ab511de0fec.png">

Reply:
More info: https://www.sarasoueidan.com/blog/accessible-icon-buttons/